kqueen_ui/asset/dynamic/js/kqueen.js

```diff
@@ -30,6 +30,10 @@ $(document).ready(function() {
     $(document.body).on("click", "a[data-toggle]", function(event) {
         location.hash = this.getAttribute("data-tabcode");
     });
+
+    // Hide flash message after 3 seconds
+    $("#flash-container").delay(3000).fadeOut(500);
```

Comment:
I would do it this way instead:
```
    setTimeout(function() {
        $(".alert.fade").alert('close');
    }, 3000);
```
It will be used for all alerts that have the fade class, so they are supposed to fade out and it will use the native Bootstrap function to close the alert, there may be other things/events that it does, so it feels better.